### DISCUSSION TITLE
crash

### DISCUSSION THREAD

**Discussion by chenfei533** [Tencent/libpag#1668] [Q&A] [answered]
有个crash，麻烦帮忙看一下
有个场景初始化了三个pagview加载本地的pag动画，这是触发次数比较多的场景，其他场景也有 不易复现

sdk版本：4.2.77
log日志：
[crash.txt](https://github.com/Tencent/libpag/files/12357810/crash.txt)

**Comment by kevingpqi123** [comment, marked as answer]
可以升级至 4.2.100, 修复过类似问题

**Reply by chenfei533** [reply]
最近发现有新的crash，以下是相关log信息，麻烦帮忙看一下
sdk版本:4.3.18 
[crash_log.txt](https://github.com/Tencent/libpag/files/12596637/crash_log.txt)
